protoc-gen-gogo/generator/generator.go

```diff
@@ -755,6 +755,14 @@ func (g *Generator) SetPackageNames() {
 
 	// Check that all files have a consistent package name and import path.
 	for _, f := range g.genFiles[1:] {
+		// "google.protobuf" pacakges all have different go package names
+		// (typepb, structpb, anypb...), so running protoc here will error.
+		// As a hack, we skip them here. It should be safe, as they packages
+		// are generated internally in this repo.
+		if *f.Package == "google.protobuf" {
+			continue
+		}
```

Comment:
FYI, here's another hack unfortunately. when running `protoc-gen-gogo` on `google.protobuf.*` stuff in this repo, I get an error with inconsistent go package names. This is expected, the new proto files are generated in separate packages: `anypb`, `durationpb`...

So the hack is to skip validation for `google.protobuf.*`. Then in the Makefile, I do `sed` to modify the package name